CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.3] - 2022-03-17
+
+### Fixed
+
+- Fixed rare errors when getting authors metadata with posts with authors with no name. [#722](https://github.com/Parsely/wp-parsely/pull/722)
```

Comment:
Consider simplifying the phrasing here if possible, it is not easy for me to understand. Also, past tense is used here while in the majority of the file we use imperative. So maybe consider writing in imperative.